llama_index/vector_stores/redis.py

```diff
@@ -103,7 +104,7 @@ def __init__(
             raise ValueError(f"Redis failed to connect: {e}")
 
         # index identifiers
-        self._prefix = index_prefix
+        self._prefix = index_prefix + "/vector"
```

Comment:
could we add this as part of the default string instead of hardcoding it here?

Reply:
Adding this as part of the default string won't resolve the issue I mentioned in the PR description, since the user can still assign some `index_prefix` that is the same or just the prefix of the namespace in the document store or index store (just like the code I given, and I think it can be a very common scenario if users don't know the story behind this).

Reply:
If you mean adding a new parameter for configuring, of course we can!